### PR TITLE
Redo README, including links and screenshot.png

### PR DESCRIPTION
**Pull request by Montandalar**
This brings screenshot.png* up to date with the current textures, and points links to right repository.

It may be a good idea to make a new GitHub release so that the badge doesn't complain about there being none. That, or use ContentDB for releases and remove/replace the badge.

*Actually it's a new file, the screenshot used prior was uploaded to GitHub as a file only, without being checked in to the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+    ██╗██████╗
+    ██║╚════██╗
+    ██║ █████╔╝
+    ██║ ╚═══██╗
+    ██║██████╔╝
+    ╚═╝╚═════╝
+```
 
-[![GitHub Release](https://img.shields.io/github/release/minetest-mods/i3.svg?style=flat)]() ![workflow](https://github.com/minetest-mods/i3/actions/workflows/luacheck.yml/badge.svg) [![ContentDB](https://content.minetest.net/packages/jp/i3/shields/downloads/)](https://content.minetest.net/packages/jp/i3/) [![PayPal](https://img.shields.io/badge/paypal-donate-yellow.svg)](https://www.paypal.me/jpg84240)
+[![GitHub Release](https://img.shields.io/github/release/mt-historical/i3.svg?style=flat)]() ![workflow](https://github.com/mt-historical/i3/actions/workflows/luacheck.yml/badge.svg) [![ContentDB](https://content.minetest.net/packages/jp/i3/shields/downloads/)](https://content.minetest.net/packages/jp/i3/)
```
Comment (luk3yx) [inline README.md:10]: Does the donate button have to be kept as a "donate to original author" or something? I don't really know

File: README.md [hunk 2]
```diff
@@ -50,8 +57,6 @@ You can also use the font size slider in the inventory, settings window.
 Thus, most chests will be unadapted to this inventory size.
 The `i3`  inventory is 9 slots wide by default, such as Minecraft.
 
-Report bugs on the [**Bug Tracker**](https://github.com/minetest-mods/i3/issues).
+Report bugs on the [**Bug Tracker**](https://github.com/mt-historical/i3/issues).
 
-**Video review on YouTube:** https://www.youtube.com/watch?v=Xd14BCdEZ3o
```
Comment (luk3yx) [inline README.md:55]: Was there any reason for removing the YouTube video? Even though it shows an older version it may still be somewhat relevant

**Review by luk3yx — COMMENTED** [review]
I missed some things

File: README.md (hunk 2 above)
Comment (Montandalar) [reply README.md:55]: @wsor4035 what do you think about the video?

File: README.md (hunk 1 above)
Comment (Montandalar) [reply README.md:10]: Removing the donate button removes the potential (however small) for someone to send money to someone who quit the mod and Minetest. I don't see what benefit there could be to sending money his way for something he doesn't maintain any more. And I don't think there's anything in the license (MIT) that would require it to stay.

File: README.md (hunk 2 above)
Comment (wsor4035) [reply README.md:55]: if where updating the rest of the media, seems related to remove/replace the video

**Pull request merged**